src/useradd.c

```diff
@@ -2420,6 +2442,7 @@ int main (int argc, char **argv)
 #endif
 	unsigned long subuid_count = 0;
 	unsigned long subgid_count = 0;
+	struct option_flags flags;
```

Comment:
Please use two spaces between the type name and the variable name.  This helps distinguish `struct option_flags` from `flags` more easily.